docs/pages/2019_MARVEL_Psik_MaX/sections/calculations.rst

```diff
@@ -263,44 +241,42 @@ the ``parameters_dict`` on a ``Dict`` model:
 Simulate submission
 ~~~~~~~~~~~~~~~~~~~
 
-At this stage, you have recreated in memory (it’s not yet stored in the
-database) the input of the graph shown bellow, whereas the outputs will
-be created later by the daemon.
+At this stage, you have created in memory (it’s not yet stored in the
+database) the input of the graph shown below. The outputs will
+be created by the daemon later on.
 
 .. figure:: include/images/verdi_graph/si/graph-full.png
    :alt:
 
-In order to check how AiiDA creates the actual input files for the
-calculation, we can perform a *dry run* of the submission process, to
-achieve that, we should modify the metadata of our builder, first
-specifying that we want a dry run, and then telling it that we don't
-want to store the provenance of this calculation (yet):
+In order to check which input files AiiDA creates, 
+we can perform a *dry run* of the submission process.
+Let's tell the builder that we want a dry run and that
+we don't want to store the provenance of the dry run:
 
 .. code:: python
 
     builder.metadata.dry_run = True
     builder.metadata.store_provenance = False
 
-After setting the appropriate metadata, is time for us to actually run
-the *dry run*, to do that we need to reach for AiiDA's run
-functionality:
+It's time run:
```

Comment:
??  Probably you meant "It is time to run" or "It's run-time"

Reply:
> Probably you meant "It is time to run"

Yes... isn't that what I wrote?

Reply:
Got it ;-)